src/genotyper.cc

```diff
@@ -719,6 +1045,14 @@ Status genotype_site(const genotyper_config& cfg, MetadataCache& cache, BCFData&
     ans->rlen = site.pos.end - site.pos.beg;
     ans->qual = 0;
 
+    // Lifted-over FORMAT fields (non-genotype based)
+    // TODO: It seems like the update command screw up the
+    // alleles (REF, ALT) fields, quick fix: update the format
+    // before updating the alleles, but KIV bug fix.
```

Comment:
This seems resolved now, it was probably due to an issue in my own memory management, where previously if I updated the FORMATs before updating GTs, some GTs will get trashed. I've moved the FORMAT to be updated after GTs now for readability, and things seems ok. Removed the misleading comments